static/src/javascripts/projects/commercial/modules/comscore.spec.ts

```diff
@@ -76,14 +92,14 @@ jest.mock('../../common/modules/commercial/commercial-features', () => ({
 describe('setupComscore', () => {
 	it('should do nothing if the comscore is disabled in commercial features', () => {
 		commercialFeatures.comscore = false;
-		setupComscore();
+		void setupComscore();
```

Comment:
I think if this returns a promise then it should be `await`-ed and the test converted to `async`?